config/initializers/sentry.rb

```diff
@@ -0,0 +1,13 @@
+Sentry.init do |config|
+  config.dsn = ENV['sentry_key']
+  config.breadcrumbs_logger = [:active_support_logger, :http_logger]
+
+  # Set tracesSampleRate to 1.0 to capture 100%
+  # of transactions for performance monitoring.
+  # We recommend adjusting this value in production
+  config.traces_sample_rate = 1.0
+  # or
```

Comment:
So which version do you prefer? ;)

Reply:
Why not bo-... Okay, missed this, thanks. ;)